connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java

```diff
@@ -236,31 +200,21 @@ public void connectorStatus() {
         assertEquals(0, taskState.id());
         assertEquals("UNASSIGNED", taskState.state());
         assertEquals(workerId, taskState.workerId());
-
-        PowerMock.verifyAll();
     }
 
     @Test
-    public void taskStatus() {
+    public void testTaskStatus() {
         ConnectorTaskId taskId = new ConnectorTaskId("connector", 0);
```

Comment:
We can use the `connector` variable instead of the literal here

Reply:
done